services/web/server/tests/unit/with_dbs/02/test_projects_nodes_handler.py

```diff
@@ -847,15 +850,48 @@ async def test_stop_node(
 
 @pytest.fixture
 def app_environment(
-    app_environment: dict[str, str], monkeypatch: MonkeyPatch
+    app_environment: dict[str, str], monkeypatch: pytest.MonkeyPatch
 ) -> dict[str, str]:
     # test_read_project_nodes_previews needs WEBSERVER_DEV_FEATURES_ENABLED=1
     new_envs = setenvs_from_dict(monkeypatch, {"WEBSERVER_DEV_FEATURES_ENABLED": "1"})
     return app_environment | new_envs
 
 
-@pytest.mark.parametrize("user_role", (UserRole.USER,))
+@pytest.fixture
+def mock_storage_api(mocker: pytest_mock.MockerFixture, faker: Faker) -> None:
```

Comment:
you are aware we do have this?
https://github.com/ITISFoundation/osparc-simcore/blob/3ff6e3086d3a0de36e526c1fa12494d784f03588/packages/pytest-simcore/src/pytest_simcore/services_api_mocks_for_aiohttp_clients.py

this would allow you to test all the way till mocked storage server